Source/Scene/createOsmBuildings.js

```diff
@@ -21,6 +21,8 @@ import Cesium3DTileStyle from "./Cesium3DTileStyle.js";
  *        specified, a default style is used which gives each building or building part a
  *        color inferred from its OpenStreetMap <code>tags</code>. If no color can be inferred,
  *        <code>options.defaultColor</code> is used.
+ * @param {Boolean} [options.showOutline=true] Whether to display the outline for models. When true, outlines \
+ *        are displayed. When false, outlines are not displayed.
```

Comment:
Maybe a slightly more tailored description for `createOsmBuildings`:  "Whether to show outlines around buildings..."